tests/integration/dns_test.py

```diff
@@ -33,7 +33,7 @@
 IPV4_DNS_NAMESERVERS = ["8.8.8.8", "1.1.1.1"]
 EXTRA_IPV4_DNS_NAMESERVER = "9.9.9.9"
 IPV6_DNS_NAMESERVERS = ["2001:4860:4860::8888", "2606:4700:4700::1111"]
-IPV6_DNS_LONG_NAMESERVER = ["0000:0000:0000:0000:0000:0000:0000:0100"]
+IPV6_DNS_LONG_NAMESERVER = ["2000:0000:0000:0000:0000:0000:0000:0100"]
```

Comment:
Why is this change required?

Reply:
Explained in commit message and also PR comment 0.

The test expect nmstate to
compress `0000:0000:0000:0000:0000:0000:0000:0100` to `::100`.

This IPv6 is IPv4-Compatible IPv6 Address(RFC 4291), so it is also
legal to show as `::0.0.1.0` which is used by rust `std::net::Ipv6Addr`.

Hence change the test IP to 2000::100 which is not ambiguous.

Reply:
ACK, thanks!